etc/requirements_ci.txt

```diff
@@ -1,38 +1,38 @@
-bcolz==0.12.1
-click==6.6
-cloudpickle==0.2.1
-Cython==0.25.1
-dask==0.10.2
-decorator==4.0.10
-Flask==0.11.1
-Flask-Cors==3.0.2
-future==0.16.0
-h5py==2.6.0
-itsdangerous==0.24
-Jinja2==2.8
-llvmlite==0.14.0
-locket==0.2.0
-MarkupSafe==0.23
-multipledispatch==0.4.9
-networkx==1.11
-numba==0.29.0  # please update the .travis.yml file!
-numexpr==2.6.1
-numpy==1.11.2  # please update the .travis.yml file!
-pandas==0.19.1  # please update the .travis.yml file!
-partd==0.3.6
-psutil==4.4.2
-psycopg2==2.6.2
-PyHive==0.2.1
-pymongo==2.9.4
-PyMySQL==0.7.9
-pytest==3.0.3
-pytest-cov==2.0.0
-python-dateutil==2.5.3
-pytz==2016.7
-pyyaml==3.11
+bcolz
```

Comment:
the point of this file was to pin the requirements we test with. Tests kept failing because downstream packages were being updated. I don't want to fix the whole world with every PR and it put a huge burden on whoever wanted to open the next PR.

Reply:
This comment still stands